README.md

```diff
@@ -3188,13 +3183,14 @@ So, it evolved into `ascli`:
 
 * 4.0.0.pre1
 
-        * now available as open source at [https://github.com/IBM/aspera-cli](https://github.com/IBM/aspera-cli) with general cleanup
-        * changed default tool name from `mlia` to `ascli`
-        * changed `aspera` command to `oncloud`
-        * changed gem name from `asperalm` to `aspera-cli`
-        * changed module name from `Asperalm` to `Aspera`
-        * removed command `folder` in `preview`, merged to `scan`
-        * persistency files go to sub folder instead of main folder
+	* now available as open source at [https://github.com/IBM/aspera-cli](https://github.com/IBM/aspera-cli) with general cleanup
+	* changed default tool name from `mlia` to `ascli`
+	* changed `aspera` command to `oncloud`
```

Comment:
I agree with @gegles on this one, maybe use something more standard? I do think `oncloud` is better than `aspera` though. CLI V3 users will be familiar with `aoc` so we could use that instead of `oncloud`.

Reply:
Oh, be careful that the file "README.md" is generated from file docs/README.erb.md
So make the changes in docs/README.erb.md , and than do "make doc".
Note the comment as first line in /README.md
Yes, I agree that "aoc" is common as abbreviation for "Aspera on Cloud".
The thin is that other products are named:
"Aspera Faspex", "Aspera Shares", ...
Also, the CLI means "Aspera Command Line" (ascli) there is already aspera in there.
other commands are "Aspera CLI Faspex Package Send"
I find it logical (I take the tone of Spock or Tuvok here) that the command line says:
"Aspera CLI OnCloud Package Send"
instead of "Aspera CLI AsperaOnCloud Package Send"
Also, other commands are not abbreviated, "faspex" is not "fx" or "afx"
Note also that any command can be abbreviated to the minimum number of character to uniquely identify the command, such as: "ascli onc", or even "ascli on"
So, it is again taste and colors, or religions, and that's not the kind of war I carry.
obviously my taste is "oncloud".
but I prefer anyway "aoc" to just "aspera" as it was in "mlia"
The legacy reason it was named "aspera", was that the first name of the application was "Aspera Files", and naming the command "AsperaFiles" with a sub application "Files" was uggly.
while "mlia aspera files browse" was more neat.
The command name was not changed when the product name changed from Aspera Files to Aspera on Cloud.
For sure in the future the name "aspera" will fade away, and some OM will rename the product again, or it will be integrated to IBM Cloud.

Reply:
@laurent-martin, I've quickly addressed the various typos as part of this PR. I was happy to do it, but just so you know, usually the idea is for the person submitting the PR to update it based on the feedback... Essentially just commit to the branch and the PR gets automatically and dynamically updated. This PR is a bit different as it still part of the rebranding effort, but in general, we try to keep the changeset for a PR relatively small and around a single feature or unit of work so that it is easier for others to review.

For this PR I will do a simple rebase onto develop and preserve all the individual commits, but often we tend to squash all the commits of a PR into a single commit that reflect a particular feature. It keeps the history cleaner and it is easier to manage.

Thanks for the explanation on the name, I now understand better where you are coming from (You may be a bit too much like Spock on this one ;-) ), although `aoc` may diverge a little bit from your naming scheme, I believe it will be clearer for the customers.

Thanks again for a the work! Keep it coming.